cdap-ui/app/cdap/components/PreviewData/DataView/Table.tsx

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React from 'react';
+import Paper from '@material-ui/core/Paper';
+import Grid from '@material-ui/core/Grid';
+import withStyles, { WithStyles, StyleRules } from '@material-ui/core/styles/withStyles';
+import isEmpty from 'lodash/isEmpty';
+import { PREVIEW_STATUS } from 'services/PreviewStatus';
+import Heading, { HeadingTypes } from 'components/Heading';
+import ThemeWrapper from 'components/ThemeWrapper';
+import VirtualScroll from 'components/VirtualScroll';
+import T from 'i18n-react';
+import classnames from 'classnames';
+
+const I18N_PREFIX = 'features.PreviewData.DataView.Table';
+
+export const messageTextStyle = {
+  fontSize: '1.3rem !important',
```

Comment:
Is it possible to use px instead of rem for the fontSize?

Reply:
It is possible, but I think rem is better for accessibility (also for consistency since this is a cherry-pick).